run_all_trials.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+TEAM_NAME=$1
+
+#TODO: get list of trials automatically
+declare -a TRIAL_NAMES=("example_trial1" "example_trial2")
+for TRIAL_NAME in ${TRIAL_NAMES[@]}; do
+  echo "Running trial: ${TRIAL_NAME}"
+  #TODO: GET THIS TO RUN THE INDIVIDUAL TRIALS CORRECTLY.
```

Comment:
in the end quoting the parameters fixed it: 2b1a20b52e46207af5b43314a1453d08ce56ef64